packages/@lwc/integration-karma/test/component/spellcheck-attribute/index.spec.js

```diff
@@ -49,6 +49,7 @@ describe('setting static values on non custom elements', () => {
         const elm = createElement('x-container', { is: Container });
         document.body.appendChild(elm);
 
+        expect(getSpellcheckValue(elm, 'div.boolean-true-v')).toBe('');
```

Comment:
This test was missing in the original test! I added it for completeness.

There are 8 cases that are being tested here:

https://github.com/salesforce/lwc/blob/7b0418c79e4159c5a8d0f8bda5ded0f1cf312ce7/packages/%40lwc/integration-karma/test/component/spellcheck-attribute/x/container/container.html#L11-L18